anta/cli/utils.py

```diff
@@ -46,27 +43,25 @@ class ExitCode(enum.IntEnum):
 
 def parse_tags(ctx: click.Context, param: Option, value: str) -> list[str] | None:
     # pylint: disable=unused-argument
-    """
-    Click option callback to parse an ANTA inventory tags
-    """
+    """Click option callback to parse an ANTA inventory tags."""
     if value is not None:
         return value.split(",") if "," in value else [value]
     return None
 
 
 def exit_with_code(ctx: click.Context) -> None:
-    """
-    Exit the Click application with an exit code.
+    """Exit the Click application with an exit code.
     This function determines the global test status to be either `unset`, `skipped`, `success` or `error`
     from the `ResultManger` instance.
     If flag `ignore_error` is set, the `error` status will be ignored in all the tests.
     If flag `ignore_status` is set, the exit code will always be 0.
     Exit the application with the following exit code:
         * 0 if `ignore_status` is `True` or global test status is `unset`, `skipped` or `success`
         * 1 if status is `failure`
-        * 2 if status is `error`
+        * 2 if status is `error`.
```

Comment:
meeeh

Reply:
Ruff formater adds periods at the end of sentences :)